app/javascript/packs/intl-tel-input.js

```diff
@@ -0,0 +1,23 @@
+// Setting hidden input for intl code
+
+const telInput = $('#user_phone_form_phone');
+const intlCode = $('#user_phone_form_international_code');
+
+// initialise plugin
+telInput.intlTelInput({
+  preferredCountries: ['us', 'ca'],
+  excludeCountries: ['as', 'ai', 'ag', 'bs', 'bb', 'bm', 'io', 'vg', 'ky', 'cx', 'cc',
+    'cw', 'dm', 'do', 'gd', 'gu', 'gg', 'im', 'jm', 'je', 'ki', 'xk', 'ms', 'nr', 'nu', 'nf', 'mp',
+    'bl', 'sh', 'kn', 'lc', 'mf', 'vc', 'sx', 'sj', 'tl', 'tk', 'tt', 'tc', 'vi', 'va', 'wf', 'eh',
+    'ax'],
```

Comment:
Can we alphabetize this list? Also, why are we excluding these countries?

Reply:
Sure,  I can alphabetize this list. 
These countries were excluded because they are countries that are not currently offered but are part of the intl-tel plugin.

Reply:
Can you please define "not currently offered"? We want to support all countries that Twilio supports for SMS. Is this a list of countries that Twilio doesn't support?

Reply:
Sure yeah, the new intl-tel plugin has more countries in the dropdown than the current live site does. 

For example, the current live site **does not** offer `Saint Kitts and Nevis` which is `+1869` but this and other countries are included as part of the intl-tel plugin.

Reply:
Thanks for the clarification. The fact that the current live site does not offer those countries is a bug, so since we're making changes to the dropdown, might as well make sure all countries we want to support are available in the dropdown. Our Twilio config has all supported countries checked, including Saint Kitts and Nevis. 

The list of countries we exclude should be based on whether or not Twilio supports it, not on what's in the current live site's dropdown. If you send me the list of countries, I will double check that they are supported in Twilio.

Reply:
Thanks @monfresh - The country list is here - https://github.com/jackocnr/intl-tel-input/blob/master/src/js/data.js